src/main/java/bio/terra/workspace/service/datareference/utils/DataReferenceValidationUtils.java

```diff
@@ -21,16 +21,15 @@ public DataReferenceValidationUtils(ObjectMapper objectMapper, DataRepoService d
   }
 
   public String validateReference(
-      DataReferenceDescription.ReferenceTypeEnum referenceType,
-      String reference,
-      AuthenticatedUserRequest userReq) {
+      ReferenceTypeEnum referenceType, String reference, AuthenticatedUserRequest userReq) {
 
-    if (referenceType.equals(DataReferenceDescription.ReferenceTypeEnum.DATAREPOSNAPSHOT)) {
+    if (referenceType != null && referenceType.equals(ReferenceTypeEnum.DATA_REPO_SNAPSHOT)) {
```

Comment:
Can we invert this? `ReferenceTypeEnum.DATA_REPO_SNAPSHOT.equals(referenceType)` would seem to imply the not-null, right?

Reply:
Nice, that works